platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -737,6 +737,25 @@ private Collection<SwirldsPlatform> createLocalPlatforms(
         return Collections.unmodifiableList(platforms);
     }
 
+    /**
+     * Create a copy of the initial signed state. There are currently data structures that become immutable after
+     * construction, and we need to make a copy to force it to be mutable again.
```

Comment:
The way this description is worded makes it sound like the data structures in question become immutable *immediately* after construction, though I assume it just means "at some point after construction"?

Is there a way to make this more clear? A reader with the wrong interpretation might wonder how making a copy will help, if the structures become immutable after being constructed

Reply:
Updated. Specifically, virtual maps become immutable after being hashed.

```
     * Create a copy of the initial signed state. There are currently data structures that become immutable after
     * being hashed, and we need to make a copy to force it to become mutable again.
```